src/user/user.service.ts

```diff
@@ -12,3 +13,17 @@ export const createUser = async (userForm: CreateUserForm): Promise<User> => {
     updated_at: Date.now().toString(),
   };
 };
+
+class UserService {
+  async findUserByEmail(email: string) {
+    return prisma.user.findUnique({ where: { email } });
+  }
+
+  async createUser(name: string, email: string, passwordHash: string) {
+    return prisma.user.create({
```

Comment:
Probably there should be some additional errors checking here.
For [ref](https://github.com/Code-the-Dream-School/ii-practicum-team-2-back/pull/10/files#diff-ca43d99639d95938d8d8969db67918c81ea24beaae5462213fd95072c6255f9aR88).

Reply:
What should I check there? I checked existing user, I checked all the fields. I did it all in controller
What else should I check here?

Reply:
Prisma exceptions. Explained in this [comment](https://github.com/Code-the-Dream-School/ii-practicum-team-2-back/pull/7/files#r2057504144) above.